src/workloads/execution/locf.yml

```diff
@@ -0,0 +1,257 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests the performance of $locf. The benchmark operations test numeric,
+  string, date and randomized data types. There is a test without partitions, with a 
+  single partition, and with multiple partitions for each data type.
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000
+    BatchSize: &batchSize 100
+    Document:
+      part1: {^Choose: {from: [1, 2, 3, 4]}}
+      part2: {^Choose: {from: [1, 2, 3, 4]}}
+      numeric: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}}, 
+        {^RandomDouble: { min: 0.0, max: 500.0 }},
+        null], weights: [1, 1, 8]}}
+      string: {^Choose: {from: [
+        {^RandomString: {length: 15}}, 
+        null], weights: [2,8]}}
+      date: {^Choose: {from: [
+        {^RandomDate: {min: "2021-01-01", max: "2023-01-01"}}, 
+        null], weights: [2, 8]}}
+      randomType: {^Choose: {from: [
+        {^RandomInt: {min: -100, max: 100}}, 
+        {^RandomDouble: { min: 0.0, max: 500.0 }}, 
+        {^RandomDate: {min: "2021-01-01", max: "2023-01-01"}}, 
+        {^RandomString: {length: 15}}, 
+        null], weights: [1, 1, 1, 1, 5]}}
+  - Nop: true
+  - Nop: true
+
+- Name: Quiesce
+  Type: QuiesceActor
+  Threads: 1
+  Database: *db
+  Phases:
+  - Nop: true
+  - Repeat: 1
+  - Nop: true
+
+- Name: locf
```

Comment:
I think actor names are supposed to be capitalized.

Reply:
done